node_monitor/node_monitor.py

```diff
@@ -95,11 +96,9 @@ def run_once(self):
                                      if event.is_actionable() and event.t2 == "DOWN"]
                 if len(events_actionable) == 0:
                     return
-                email = NodeMonitorEmail(
-                    "\n\n".join(str(event) for event in events_actionable)
-                    + "\n\n" + self.stats_message()
-                )
-                email.send_recipients(emailRecipients)
+                message = "\n\n".join(
+                    str(event) for event in events_actionable) + "\n\n" + self.stats_message()
+                self.send_to_subscribers(message)
```

Comment:
This is a bit hard to read, we should move it so that the string is concatenated on multiple lines.

Something like 
```python
message = (
    "\n\n".join(str(i) for i in range(10))
    + "\n\n"
    + str(4)
)

print(message)
```

Reply:
Please see the latest commit. Is this better?